wallet-access/src/App.vue

```diff
@@ -1,21 +1,27 @@
+<template>
+  <Suspense>
+    <template #fallback>
+      Loading...
+    </template>
+    <template #default>
+      <div>
+        {{ address }}
+      </div>
+    </template>
+  </Suspense>
+</template>
+
 <script setup lang="ts">
 import { MetamaskProvider, createProvider, ChainTypes } from '@rarimo/provider'
 
-const getMetamaskWalletInfo = async () => {
-  // Connect to the Metamask wallet in the browser using Web3.js, using the MetamaskProvider interface to limit bundle size.
-  const provider = await createProvider(MetamaskProvider)
-  await provider.connect()
+// Connect to the Metamask wallet in the browser using Web3.js, using the MetamaskProvider interface to limit bundle size.
+const provider = await createProvider(MetamaskProvider)
+await provider.connect()
 
-  // Get the address of the wallet
-  console.log('Address:', provider.address)
-  console.log('chainType:', ChainTypes[provider.chainType!])
-  console.log('providerType:', provider.providerType)
-}
+// Get the address of the wallet and other info
+console.log('Address:', provider.address)
+console.log('chainType:', ChainTypes[provider.chainType!])
+console.log('providerType:', provider.providerType)
 
-getMetamaskWalletInfo()
```

Comment:
bring back this, and remove suspense, it's experimental feature which not recommended to use in the production